packages/editor-ui/src/utils.ts

```diff
@@ -1,3 +1,5 @@
+import sanitizeHtmlModule from 'sanitize-html';
```

Comment:
can we use xss library that we already use elsewhere?

Reply:
Good point, updated.